borg/selftest.py

```diff
@@ -0,0 +1,63 @@
+import sys
+import time
+from unittest import TestResult, TestSuite, defaultTestLoader
```

Comment:
then these test cases maybe should have a comment that they must stay with unittest because of borg.selftest. and they also need to get moved to a module that does not import py.test or it will fail on users' systems.